### PR TITLE
Set up skeleton for tab graph window using old UI system 

### PR DESCRIPTION
**Pull request by FelixJons**

### CODE REVIEW

File: TrafficSimulator/Assets/DraggableUI.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    private Vector2 _offset;
+
+    public void OnBeginDrag(PointerEventData eventData)
+    {
+        _offset = eventData.position - new Vector2(transform.position.x, transform.position.y);
+    }
+
+    public void OnDrag(PointerEventData eventData)
+    {
+        if (Mouse.current.leftButton.isPressed)
```
Comment (marc7s) [inline TrafficSimulator/Assets/DraggableUI.cs:16]: Remove curly brackets on oneliner

File: TrafficSimulator/Assets/GraphChartFeed.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                if (i == 0)
+                    graph.DataSource.SetCurveInitialPoint("Player 2",i*5, Random.value * 10f + 10f);
+                else
+                    graph.DataSource.AddLinearCurveToCategory("Player 2", 
+                        new DoubleVector2(i*5 , Random.value * 10f + 10f));
+            }
+            graph.DataSource.MakeCurveCategorySmooth("Player 2");
+            graph.DataSource.EndBatch();
+        }
+        // StartCoroutine(ClearAll());
```
Comment (marc7s) [inline TrafficSimulator/Assets/GraphChartFeed.cs:31]: Can this comment be removed?

File: TrafficSimulator/Assets/GraphChartFeed.cs [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            
+            graph.Scrollable = false;
+            graph.HorizontalValueToStringMap[0.0] = "Zero"; // example of how to set custom axis strings
+            graph.DataSource.StartBatch();
+            graph.DataSource.ClearCategory("Player 1");
+            graph.DataSource.ClearAndMakeBezierCurve("Player 2");
+            
+            for (int i = 0; i <5; i++)
+            {
+                graph.DataSource.AddPointToCategory("Player 1",i*5,Random.value*10f + 20f);
```
Comment (marc7s) [inline TrafficSimulator/Assets/GraphChartFeed.cs:21]:
```suggestion
                graph.DataSource.AddPointToCategory("Player 1", i*5, Random.value * 10f + 20f);
```

File: TrafficSimulator/Assets/GraphChartFeed.cs [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+                if (i == 0)
+                    graph.DataSource.SetCurveInitialPoint("Player 2",i*5, Random.value * 10f + 10f);
+                else
+                    graph.DataSource.AddLinearCurveToCategory("Player 2", 
+                        new DoubleVector2(i*5 , Random.value * 10f + 10f));
```
Comment (marc7s) [inline TrafficSimulator/Assets/GraphChartFeed.cs:26]:
```suggestion
                        new DoubleVector2(i*5, Random.value * 10f + 10f));
```

File: TrafficSimulator/Assets/UI/OverlayController.cs [hunk 5]
```diff
@@ -242,10 +242,14 @@ private void StatisticsButtonOnClicked()
             
         }
 
+        public GameObject WorldWindowStatistic;        
+        
         private void WorldOptionButtonOnClicked()
         {
             _isWorldOpen = !_isWorldOpen;
-            _worldUI.visible = _isWorldOpen;
+            //_worldUI.visible = _isWorldOpen;
```
Comment (marc7s) [inline TrafficSimulator/Assets/UI/OverlayController.cs:250]: Can this comment be removed?

**Review by marc7s — CHANGES_REQUESTED** [review]

File: TrafficSimulator/Assets/GraphChartFeed.cs (hunk 2 above)
Comment (FelixJons) [reply TrafficSimulator/Assets/GraphChartFeed.cs:31]: Leave this for now, will change when finished setting up everything in this window.

File: TrafficSimulator/Assets/DraggableUI.cs (hunk 1 above)
Comment (FelixJons) [reply TrafficSimulator/Assets/DraggableUI.cs:16]: Fixed

File: TrafficSimulator/Assets/GraphChartFeed.cs (hunk 3 above)
Comment (FelixJons) [reply TrafficSimulator/Assets/GraphChartFeed.cs:21]: Leave this for now, will change when finished setting up everything in this window.

File: TrafficSimulator/Assets/GraphChartFeed.cs [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+            graph.HorizontalValueToStringMap[0.0] = "Zero"; // example of how to set custom axis strings
+            graph.DataSource.StartBatch();
+            graph.DataSource.ClearCategory("Player 1");
+            graph.DataSource.ClearAndMakeBezierCurve("Player 2");
+            
+            for (int i = 0; i <5; i++)
+            {
+                graph.DataSource.AddPointToCategory("Player 1",i*5,Random.value*10f + 20f);
+                if (i == 0)
+                    graph.DataSource.SetCurveInitialPoint("Player 2",i*5, Random.value * 10f + 10f);
```
[the remark this answers is not in the document]
Comment (FelixJons) [reply TrafficSimulator/Assets/GraphChartFeed.cs:23]: Leave this for now, will change when finished setting up everything in this window.

File: TrafficSimulator/Assets/GraphChartFeed.cs (hunk 4 above)
Comment (FelixJons) [reply TrafficSimulator/Assets/GraphChartFeed.cs:26]: Leave this for now, will change when finished setting up everything in this window.

File: TrafficSimulator/Assets/UI/OverlayController.cs (hunk 5 above)
Comment (FelixJons) [reply TrafficSimulator/Assets/UI/OverlayController.cs:250]: Fixed

**Review by marc7s — APPROVED** [review]

**Pull request merged**